### ISSUE TITLE
make h5 handler default for nc4 files

### ISSUE THREAD

**Issue by Mikejmnez** [OPENDAP/bes#1002]
## Problem

I want to server .nc4 dataset with DAP4 types such as Int64s. By default in the `hyrax:snapshot`, the nc4 files is assigned to the netCDF handler. When I try to inspect the dmr of the .nc4 file I get an Hyrax 500 error. To actually serve this nc4 file I need to assign h5 handler via the `site.conf` file  (see https://github.com/OPENDAP/hyrax_guide/issues/17#issue-2676571699). This is less than ideal.

## (potential) solution
 
Assign the h5 data handler to any .nc4 file by default on the `bes.conf` file. I think this should work since netCDF4 is a subset of h5...

**Comment by Mikejmnez** [comment]
> fix the netcdf_handler to actually support netcdf-4?

Yes - I think your proposed solution is better, and likely more involved (although that I am not sure).

For now, I am including an example on the `Hyrax Guide` on serving `nc4`  enabled by  h5 handler via the `site.conf` file.

**Comment by jgallagher59701** [comment]
This is probably obvious, but I'll say it for completeness sake, the TypeMatch values above are regular expressions and are applied to the whole path for the file, so if all the netCDF4 files are in one subtree and all the netCDF3 files in another, the fact they end in the same extension is not a problem for a particular server installation.

Also, while regexes with ORs are possible, the OR operator is a performance killer. For a regex with two or more OR clauses, think nested loops.
